tests/baselines/reference/user/enhanced-resolve.log

```diff
@@ -1,5 +1,10 @@
 Exit Code: 1
 Standard output:
+node_modules/enhanced-resolve/lib/CachedInputFileSystem.js(120,20): error TS2532: Object is possibly 'undefined'.
```

Comment:
I guess.. `pop` now returns `T | undefined` instead of just `T`?

Reply:
nope, previously `this.levels: any[]`, now it's `this.levels: Set<any>[]`, so `pop` now returns `Set<any> | undefined` not `any | undefined ==> any`.